### PR TITLE
Content Updates

### PR DESCRIPTION
**Pull request by aphelionz**
This PR:

- Adds better verbiage above each table
- Changes the ID and Version rows to complete
- Ecosystem updates

image

### CODE REVIEW

**Comment by koivunej** [comment]
I wonder if we should be adding text to describe the real status (like bitswap is probably at go-ipfs 0.4 level or slower).

RE: Add: Turns out it never was even a single block file support and it's gone now. So we don't have a nice API but we have the means to do "add" as shown in ipfs-http.

RE: Pin: I'd remove that `Yes` and place `Initial` or something to it as it's only in-memory, remove the http "In PR"; it's basically missing.

**Comment by aphelionz** [comment]
No problem. Two questions:

1. what should I put for add
2. Should we call bitswap "partial" i.e. yellow based on that?

**Comment by koivunej** [comment]
For `add` ... Yeah I don't know yet. We are trying to be a library so we don't have everything ready to go but we are trying to have tools to go everywhere yourself. How about ... "No"? I mean there definitely could be a way now that we switched to tokio.

Well, if we define go-ipfs 0.5 as "complete" then we are most definitely partial. I think this definition makes sense.

**Comment by aphelionz** [comment]
Pushed those updates

File: index.html [hunk 1]
```diff
@@ -190,9 +199,32 @@ <h2>Remaining Work</h2>
...
+          is an implementation that focuses on the suite of conformance
+          tests provided by Protocol Labs
+        </li>
+        <li>
+          <a href="#">rust-ipfs-api</a> - A HTTP client for an existing
+          go-ipfs or js-ipfs node. Supports both hyper and actix
+        </li>
+        <li>
+          <a href="#">ipfs-embed</a> - A light client implementation
+          based on sled
```
Comment (koivunej) [inline index.html]:
```suggestion
          <a href="#">ipfs-embed</a> - A light client implementation
          with a <a href="https://github.com/spacejam/sled">sled</a> backend
```

I did not check if this is accurate but I think the wording is now better?

**Review by koivunej — APPROVED** [review]
lets change it when we find discrepencies.

**Pull request merged**